src/hotspot/share/opto/phase.hpp

```diff
@@ -118,7 +121,9 @@ class Phase : public StackObj {
     f(     _t_fillBuffer,            "fill buffer")              \
     f(     _t_registerMethod,        "install_code")             \
     f(   _t_temporaryTimer1,         "tempTimer1")               \
-    f(   _t_temporaryTimer2,         "tempTimer2")
+    f(   _t_temporaryTimer2,         "tempTimer2")               \
+    f(   _t_testTimer1,              "testTimer1")               \
```

Comment:
Would `_t_testPhase1` and `_t_testPhase2` be a better name?

Reply:
okay